network_modeling/configs/templates/1.5/aruba/common/acl.j2

```diff
@@ -75,4 +75,5 @@ access-list ip cmn-can
 {%- endif %}
 {% set sequence = sequence+10 %}    {{ sequence }} permit any any any
 apply access-list ip mgmt control-plane vrf default
+apply access-list ip mgmt control-plane vrf CSM
```

Comment:
Should this be?
```suggestion
apply access-list ip mgmt control-plane vrf {{ variables.VRF }}
```

Reply:
Also should line 276(ish) in `canu/generate/switch/config/config.py` have its default capitalized?